cfd_protocol/src/lib.rs

```diff
@@ -22,6 +22,9 @@ use std::collections::HashMap;
 /// In satoshi per vbyte.
 const MIN_RELAY_FEE: u64 = 1;
 
+// TODO use Amount type
+const P2PKH_DUST_LIMIT: u64 = 546;
```

Comment:
I thin the recent rust-bitcoin version has a constant for this!

Reply:
I think Philipp tackled this in a1cc6badb21de442403f4f8fb046b139076d498e.